sdk/sphere/azure-resourcemanager-sphere/src/test/java/com/azure/resourcemanager/sphere/AzureSphereManagerTests.java

```diff
@@ -0,0 +1,95 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.resourcemanager.sphere;
+
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.management.AzureEnvironment;
+import com.azure.core.management.Region;
+import com.azure.core.management.profile.AzureProfile;
+import com.azure.core.test.TestBase;
+import com.azure.core.test.annotation.LiveOnly;
+import com.azure.core.util.Configuration;
+import com.azure.core.util.CoreUtils;
+import com.azure.identity.DefaultAzureCredentialBuilder;
+import com.azure.resourcemanager.resources.ResourceManager;
+import com.azure.resourcemanager.sphere.models.Catalog;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.util.Random;
+
+public class AzureSphereManagerTests extends TestBase {
+    private static final Random RANDOM = new Random();
+    private static final Region REGION = Region.US_EAST;
+    private String resourceGroupName = "rg" + randomPadding();
+    private AzureSphereManager azureSphereManager = null;
+    private ResourceManager resourceManager;
+    private boolean testEnv;
+
+    @Override
+    public void beforeTest() {
+        final TokenCredential credential = new DefaultAzureCredentialBuilder().build();
+        final AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE);
+
+        azureSphereManager = AzureSphereManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile);
+
+        resourceManager = ResourceManager
+            .configure()
+            .withLogOptions(new HttpLogOptions().setLogLevel(HttpLogDetailLevel.BASIC))
+            .authenticate(credential, profile)
+            .withDefaultSubscription();
+
+        // use AZURE_RESOURCE_GROUP_NAME if run in LIVE CI
+        String testResourceGroup = Configuration.getGlobalConfiguration().get("AZURE_RESOURCE_GROUP_NAME");
+        testEnv = !CoreUtils.isNullOrEmpty(testResourceGroup);
+        if (testEnv) {
+            resourceGroupName = testResourceGroup;
+        } else {
+            resourceManager.resourceGroups()
+                .define(resourceGroupName)
+                .withRegion(REGION)
+                .create();
+        }
+    }
+
+    @Override
+    protected void afterTest() {
+        if (!testEnv) {
+            resourceManager.resourceGroups().beginDeleteByName(resourceGroupName);
+        }
+    }
+
+    @Test
+    @LiveOnly
+    public void testCreateCatalog() {
+        Catalog catalog = null;
+        try {
+            String catalogName = "catalog" + randomPadding();
+            // @embedStart
+            catalog = azureSphereManager.catalogs()
+                .define(catalogName)
+                .withRegion(Region.create("global", "Global"))
```

Comment:
Define it at `private static final Region REGION`

Reply:
Fixed in the new version.